### PR TITLE
fixed git checkout and version logic

### PR DESCRIPTION
**Pull request by adammhaile**
added ahaile to job emails

### CODE REVIEW

File: build-scripts/ose_images/ose_images.sh [hunk 1]
```diff
@@ -332,11 +331,12 @@ setup_git_repo() {
     echo " git_path: ${git_path} "
     echo " git_branch: ${git_branch} "
   fi
+  set -e
```
Comment (stevekuznetsov) [inline build-scripts/ose_images/ose_images.sh]: Why?

Comment (adammhaile) [inline build-scripts/ose_images/ose_images.sh]: So that it will always hard fail if the below commands fail.

Comment (stevekuznetsov) [inline build-scripts/ose_images/ose_images.sh]: Oh wow I read this backwards... Amazed we don't have it on already :/

**Comment by adammhaile** [comment]
@jupierce

File: build-scripts/ose_images/ose.conf [hunk 2]
```diff
@@ -113,7 +113,7 @@ declare -A dict_git_compare
 # Variables used for comparing git repos
 # format: git_repo git_path git_Dockerfile_name matching_style
 dict_git_compare['aos-f5-router-docker']="${BASE_GIT_REPO} ose/images/router/f5 Dockerfile match_git"
-if [ ${MAJOR_RELEASE} == "3.2" ] || [ ${MAJOR_RELEASE} == "3.3" ]  || [ ${MAJOR_RELEASE} == "3.4" ]; then
+if [ ${MAJOR_RELEASE} == "3.2" ] || [ ${MAJOR_RELEASE} == "3.3" ]  || [ ${MAJOR_RELEASE} == "3.4" ] || [ ${MAJOR_RELEASE} == "3.5" ]; then
```
Comment (kargakis) [inline build-scripts/ose_images/ose.conf]: `${MAJOR_RELEASE} =~ 3.[2-5]`?

**Pull request merged**